index.html

```diff
@@ -150,5 +150,36 @@ <h3 class="framework-text">Skills</h3>
                 <li>Terminal</li>
             </ul>
         </section>
+        <section>
+            <p class="text-form">
+                I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+            </p>
+            <form>
+                <label for = "name"></label>
+                <input type="text" id="name" class="input-padron" required placeholder="Full name">
+                <hr>
+                
+                <label for="mail"></label>
+                <input type="email" id="mail" class="input-padron" required placeholder="email address">
+                <hr>
+    
+                <label for="msj"></label>
+                <textarea rows="4" cols="50" maxlength="500" id="msj" class="input-padron" placeholder="Enter text here"></textarea>
+                <button class="form-buttom" type="submit">Get in touch</button>
+            </form>
+            <hr>
```

Comment:
- [ ] KIndly make sure you add the button style link `hover` `active` to this submit button to increase the user experience, Also it could be better if you add `cursor: pointer`  so the user already now that this button is clickable :+1: 

![Screenshot from 2023-02-24 09-10-30](https://user-images.githubusercontent.com/57408419/221116629-13ae6f81-145a-4b94-ad7f-e6a32d5b4fdb.png)